src/cbor_encode.c

```diff
@@ -14,24 +14,30 @@
 _Static_assert((sizeof(size_t) == sizeof(void *)),
 	"This code needs size_t to be the same length as pointers.");
 
-uint8_t get_additional(uint32_t len, uint8_t value0)
-{
-	switch(len) {
-		case 0: return value0;
-		case 1: return 24;
-		case 2: return 25;
-		case 3: return 25;
-		case 4: return 26;
-		case 5: return 26;
-		case 6: return 26;
-		case 7: return 26;
-		case 8: return 27;
+static uint8_t log2ceil(uint32_t val)
+{
+	switch(val) {
+		case 1: return 0;
+		case 2: return 1;
+		case 3: return 2;
+		case 4: return 2;
+#if 0 /* 64-bit numbers are not supported */
```

Comment:
Should we have this behind a config or other define? Keeping `if 0` here looks like committing dead/debug code, even if the comment explains the intent.

Reply:
Eventually it could be behind a define, but right now, every number everywhere is 32 bits, so 64-bit support is not happening in the near future, so I'll remove it instead.